src/include/common/container/bitmap.h

```diff
@@ -42,16 +42,19 @@ class RawBitmap {
   // Always reinterpret_cast from raw memory.
   RawBitmap() = delete;
   ~RawBitmap() = delete;
+  /**
```

Comment:
Does doxygen complain about this?
If so, maybe it's better to add this generic comment into the macro.